### DISCUSSION TITLE
v2.6.3

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#67] [Announcements]
## What's Changed
* fix: use weights_only for loading by @kit1980 in https://github.com/lllyasviel/Fooocus/pull/3427
* fix/feat: add checkbox and config to disable updating selected styles when describing an image by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3430

## New Contributors
* @kit1980 made their first contribution in https://github.com/lllyasviel/Fooocus/pull/3427

**Full Changelog**: https://github.com/mashb1t/Fooocus/compare/v2.6.2...v2.6.3

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.6.3'>v2.6.3</a>.</em>

**Comment by soyv4** [comment]
Please tell me how to fix the error
![2024-8-6 0-20-11](https://github.com/user-attachments/assets/e6e622f4-b166-4143-bcd1-9188f25d0222)

**Reply by mashb1t** [reply]
Do not use value string "None" but None (see [here](https://github.com/mashb1t/Fooocus/blob/main/modules/config.py#L523), do only use values of [here](https://github.com/mashb1t/Fooocus/blob/main/modules/flags.py#L89) as sam model, e.g. "vit_l" ("vit_b" is already the default)

**Reply by soyv4** [reply]
> Не используйте строку значений "None", а None (см. [здесь](https://github.com/mashb1t/Fooocus/blob/main/modules/config.py#L523) , используйте только значения [here](https://github.com/mashb1t/Fooocus/blob/main/modules/flags.py#L89) в качестве модели sam, например "vit_l" ("vit_b" уже является значением по умолчанию)

![image](https://github.com/user-attachments/assets/9405eb22-1af3-471b-97aa-c6d6c92efbeb)
I change null to None, an error of incorrect configuration comes out, I don't understand where to fix it?

**Reply by mashb1t** [reply]
You do not need to provide None in the config for ip_image as this is the default anyways. Follow the 4 steps in your screenshot below to ensure your file is correctly formatted, especially line 124.

**Reply by soyv4** [reply]
> Вам не нужно указывать None в конфигурации для ip_image, так как это значение по умолчанию в любом случае. Выполните 4 шага на скриншоте ниже, чтобы убедиться, что ваш файл правильно отформатирован, особенно строка 124.

I haven't changed anything, could you fix the configuration file so that I can understand what needs to be written?
[config.txt](https://github.com/user-attachments/files/16519542/config.txt)

**Reply by mashb1t** [reply]
No. Open the file, adjust line 124 as mentioned below with the cirrect value or delete these ip_image entries as they already use the default. 
You can find the correct default value you have to use here in the example config file.

**Reply by soyv4** [reply]
> Нет. Откройте файл, настройте строку 124, как указано ниже, с правильным значением или удалите эти записи ip_image, поскольку они уже используют значение по умолчанию. Вы можете найти правильное значение по умолчанию, которое вам нужно использовать, здесь, в примере файла конфигурации.

![image](https://github.com/user-attachments/assets/34b6c996-e786-4909-92c4-de9413d3a20e)
Did I understand correctly that I need to delete these lines?

**Reply by mashb1t** [reply]
Either delete the lines or use None instead of null (not "None", but None or none without doublequotes).
Why did you add them in the first place?

**Reply by soyv4** [reply]
> Нет. Откройте файл, настройте строку 124, как указано ниже, с правильным значением или удалите эти записи ip_image, поскольку они уже используют значение по умолчанию. Вы можете найти правильное значение по умолчанию, которое вам нужно использовать, здесь, в примере файла конфигурации.

![image](https://github.com/user-attachments/assets/73da41df-506f-4ffd-9450-0c4b970ce85c)
Now this error, why is this happening with default settings ?

**Reply by soyv4** [reply]
> Либо удалите строки, либо используйте None вместо null (не "None", а None или none без двойных кавычек). Зачем вы их вообще добавили?

I copied this from the config_modification_tutorial text document into the config

**Reply by soyv4** [reply]
> Либо удалите строки, либо используйте None вместо null (не "None", а None или none без двойных кавычек). Зачем вы их вообще добавили?

The problem was not solved

**Reply by soyv4** [reply]
> Либо удалите строки, либо используйте None вместо null (не "None", а None или none без двойных кавычек). Зачем вы их вообще добавили?

I need to copy the text from config_modification_tutorial and put the config in the file, do I understand correctly? why does this error occur by default then? please send me a config with the correct parameters so that I can understand how it should look.

**Reply by mashb1t** [reply]
You can just remove those lines for ip_image_* and it works. I've double-checked, currently empty values such as "None", None or null do not work but this might be fixed in one of the next releases.

**Reply by mashb1t** [reply]
@soyv4 fixed in main, please restart Fooocus. You can now use "None", None or "" as values. See https://github.com/mashb1t/Fooocus/pull/70

**Reply by soyv4** [reply]
> @soyv4исправлено в main, перезапустите Fooocus. Теперь вы можете использовать "None", None или "" в качестве значений. См. #70

thanks

**Reply by soyv4** [reply]
> @soyv4исправлено в main, перезапустите Fooocus. Теперь вы можете использовать "None", None или "" в качестве значений. См. #70

![image](https://github.com/user-attachments/assets/58a780ec-ff22-4f80-971c-ccea0296b781)
The error still remained - The system could not correctly interpret the value of "vit_b" for the default_inpaint_mask_sam_model key, but continued working using the same default value.

**Reply by mashb1t** [reply]
true, thanks, fixed in https://github.com/mashb1t/Fooocus/pull/71

**Comment by neurounderground** [comment]
when will you add "flux" support?

**Reply by mashb1t** [reply]
@neurounderground no plans, see https://github.com/lllyasviel/Fooocus/issues/3424#issuecomment-2265401271
